tests/navigation.test.ts

```diff
@@ -7,19 +7,12 @@ test('Test: Navigation', async ({ page, baseURL }) => {
     await expect(page).toHaveTitle("Opencast Editor");
 
     // checks if Navbar on left has 4 elements
-    const locator = page.locator('#root > div > div > nav > li');
-    await expect(locator).toHaveCount(4);
-
-    // Navigation to Metadata
-    await page.click('li[role="menuitem"]:has-text("Metadata")');
-
-    // Navigation to Tracks
-    await page.click('li[role="menuitem"]:has-text("Tracks")');
+    const length = await page.locator('#root > div > div > nav > li').count();
```

Comment:
Would it make things easier to add an HTML attribute like `data-testid="mainNavButton"` for selection instead of navigating through the whole DOM tree?

Reply:
Probably would be more robust. On the other hand, I'm not sure about introducing ids just for testing. Is that common practice?